scripts/publish.js

```diff
@@ -35,6 +35,7 @@ var tagName = [moduleName, newVersion].join('-');
 
 if (moduleName === 'google-cloud') {
   tagName = ('v' + newVersion);
+  cp(path.join(__dirname, '../README.md'), cwd);
```

Comment:
The README needs to be there to be included in the tarball published to npm. Should I include it in gitignore?